pkgs/development/embedded/nmrpflash/default.nix

```
@@ -10,10 +10,10 @@ stdenv.mkDerivation rec {
   version = "0.9.22";
 
   src = fetchFromGitHub {
-    owner  = "jclehner";
-    repo   = "nmrpflash";
-    rev    = "v${version}";
-    sha256 = "sha256-gr/7tZYnuXFvfIUh2MmtgSbFoELTomQ4h05y/WFDhjo=";
+    owner = "jclehner";
+    repo = pname;
```

```suggestion
    repo = "nmrpflash";
```

The old pattern is actually preferred: `pname` is rather arbitrary, whereas `repo` is not.